### DISCUSSION TITLE
Architecture Community [2024-12-09]

### DISCUSSION THREAD

**Discussion by qor-lb** [eclipse-score/score#52] [Architecture Community]
# Participants
- Markus Bechter @ramceb 
- Nico Hartmann @HartmannNico 
- Fabian Frie 
- Lars Bauhofer @qor-lb 
- Zoran Cutura 
- Arvid Sievert 
- Gerd Schäfer 
- Martin Thiede @mthiede-acn2 
- Alexander Warg

# Agenda
1. Approval meeting minutes #3 
2. Next steps agreed in #3
   1. Roadmap SCORE MVP -> Translation of Roadmap into GitHub issues 
      - MS Q1/25 Concepts agreed & implemented (incubation repository without infra & process adherence)
      - MS Q2/25 Feature complete (partial contribution request incl. Req & Arch & Code - experimental feature in main)
      - MS Q?/25 Certifiable (first version of the MVP fulfilling the required KPIs for the certification)
   2. Creation of Topic Teams & Preparation Incubation Repos
      - Fixed Execution Order Team
      - Orchestration Team
      - IPC Team
      - Logging Team
      - Persistency Team
   3. Definition of participants
      - Operational: 13.01.2025
      - PL/Facilitator per Team nominated
      - Participants agreed/contribution committed
 3. Initial Setup of Score Repo (https://github.com/orgs/eclipse-score/projects/1/views/1)
    - #24 
    - #25
4. Next F2F architecture community

# Meeting minutes
to 1. 
- Content approved - summary of agreed outcomes in separate section (avoid reading of discussion thread) --> @qor-lb 

to 2.1 
- MS Certifiable to be defined by the respective cross functional teams --> @qor-lb 

to 2.2 
- Combination of FEO & Orchestration Team - creation of discussion per team + incubation repo + github team --> @johannes-esr 

to 2.3 
- case by case decision for dedicated PL per Team or organization within the team
- participants to be defined until 16.12. + nomination of facilitator (majority vote for facilitator if multiple choices exist)

to 3. 
- PR in the main SCORE repo for the parts relevant for MVP --> @ramceb / @qor-lb 

to 4. 
- Planning of workshops for the next year (every 2 months, e.g. PI Planning) --> Agreement in TL Circle for meeting structure between teams

**Comment by HartmannNico** [comment]
I struggle and object to combine FEO and Orchestration in one repo. Of course there is orchestration in FEO, but it is an application framework and not a core framework. Hence there should be an explicit core framework for Orchestration, having it's own repo. FEO should utilize that core framework. That was also one outcome of the discussion in the Architecture workshop last week.
